modules/identity/pages/super-admin-api-permissions.adoc

```diff
@@ -180,4 +180,4 @@ These are additional REST APIs that you have access to when you are using the Su
 
 === Additional accessible APIs
 * GET
-** platform/license
+** system/information
```

Comment:
@abirembaut am I wrong with this change?

Reply:
@abirembaut up please

Reply:
:heavy_check_mark: